docs/blog/gatekeeper-policies-as-oci-image.md

```diff
@@ -0,0 +1,215 @@
+# Bundle with ORAS, test with `gator` and deploy with GitOps your Gatekeeper policies as OCI image
+
+_Author: [Mathieu Benoit](https://www.linkedin.com/in/mathieubenoitqc/), DevRel Engineer, Google_
+
+[ORAS](https://oras.land/) is an important tool out there for working with OCI artifacts and OCI registries. As one of the users and advocates of ORAS, I witnessed the [growing trend in both user adoption and contributions in 2022](https://oras.land/blog/oras-looking-back-at-2022-and-forward-to-2023/). I thought I will share via this blog, an end-to-end scenario with OPA Gatekeeper policies and ORAS, from bundling to deploying them.
```

Comment:
It would be appropriate to tell the reader what ORAS is - I can almost guarantee someone will come to the page and now know. E.g,. "ORAS "OCI Registry as Storage"

Reply:
Done